testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -566,3 +566,20 @@ def test_alignto_reorder_atomgroups():
     ref = u.atoms[[3, 2, 1, 0]]
     rmsd = align.alignto(mobile, ref, select='bynum 1-4')
     assert_allclose(rmsd, (0.0, 0.0))
+
+
+def test_alignto_same_universe_deprecation_warning():
+    # Alike Issue 1825
+    u = mda.Universe(GRO_MEMPROT)
+    # In my case, two selections of a GRO_MEMPROT lipid
+    mobile = u.atoms.residues[4].atoms
+    ref = u.atoms.residues[12].atoms
+
+    wmsg = ("The reference and mobile selections are atom groups"
+            "from the same universe, the reference selection will" 
+            "be transformed. If it is not the behavior you intend," 
```

Comment:
```suggestion
            "be transformed. If it is not the behavior you intend,"
```

Reply:
> These are the problematic whitespaces

thanks a lot! I did notice the whitespace outside the string 🙌🏼